tests/telemetry/test_client.py

```diff
@@ -801,7 +801,7 @@ def test_databricks_tracking_uri_scheme(mock_requests, tracking_uri_scheme, term
 
 
 @pytest.mark.no_mock_requests_get
-def test_disable_events(mock_requests):
+def test_disable_events(mock_requests, bypass_env_check):
```

Comment:
We don't need `bypass_env_check` in this test :)

Reply:
Without specifying it, it fails locally. I tested it 100 times too... 100% failure rate without it. It's defintely something weird with mocking that client config. Although there's a completely different additional test that's failing when run in CI... I think there's some weird race condition going on.